watching only wallet: Asked for password to discover accounts.

This must be due to something recent. When creating a watching only wallet I am asked to provide a passphrase which does not exist to discover accounts that can't be discovered. Afterwards I am unable to open the wallet at all with the error: 
```
react_devtools_backend.js:6 Error: 12 UNIMPLEMENTED: wallet.Unlock: watching only wallet: watching wallets can not be unlocked
```

![discoveraccounts](https://user-images.githubusercontent.com/34998433/97861647-718ece80-1d47-11eb-947b-c92992ab42d1.png)

--- a/app/components/views/GetStartedPage/hooks.js
+++ b/app/components/views/GetStartedPage/hooks.js
@@ -145,12 +145,13 @@ export const useGetStarted = () => {
         console.log("is At Start Wallet");
         const { selectedWallet } = context;
         const { passPhrase } = context;
+        const { isWatchingOnly, isTrezor } = selectedWallet.value;
         const hasPassPhrase = !!passPhrase;
         onStartWallet(selectedWallet, hasPassPhrase)
           .then((discoverAccountsComplete) => {
             setSelectedWallet(selectedWallet);
             const { passPhrase } = context;
-            if (!discoverAccountsComplete && !passPhrase) {
+            if (!discoverAccountsComplete && !passPhrase && !isWatchingOnly && !isTrezor) {
               // Need to discover accounts and the passphrase isn't stored in
               // context, so ask for the private passphrase before continuing.
               send({ type: "WALLET_DISCOVERACCOUNTS_PASS" });

